yt_dlp/extractor/twitter.py

```diff
@@ -1585,6 +1585,9 @@ def _real_extract(self, url):
         if not broadcast:
             raise ExtractorError('Broadcast no longer exists', expected=True)
         info = self._parse_broadcast_data(broadcast, broadcast_id)
+        info['title'] = broadcast.get('status')
```

Comment:
imo there should be a fallback here. For the test case the title would be `None`.